app/scripts/modules/core/src/application/modal/PermissionsConfigurer.tsx

```diff
@@ -188,8 +192,9 @@ export class PermissionsConfigurer extends React.Component<IPermissionsConfigure
     return (
       <div className="permissions-configurer">
         {this.state.permissionRows.map((row, i) => {
-          const permissionTypeLabel = PermissionsConfigurer.accessTypes.find(type => type.value === row.access).label;
-
+          const permissionTypeLabel = []
+            .concat(PermissionsConfigurer.accessTypes, PermissionsConfigurer.legacyAccessTypes)
```

Comment:
nit: I think the es6 syntax is a little cleaner here but what do you think? `[...PermissionsConfigurer.accessTypes, ...PermissionsConfigurer.legacyAccessTypes]`

Reply:
I concur! Thanks!